api/generic-cti/newcall-event.rst

```diff
@@ -0,0 +1,264 @@
+               }
+
+         Response:
+
+            .. code-block:: json
+
+               {}
+
+         Sample curl command:
+
+            .. code-block:: sh
+
+               $ curl --request POST 'https://mh-docs-dev.zammad.com/api/v1/cti/PhXp3ZG63QyJfcb1pVOnBKJWIbI' \
+                  --header 'Content-Type: application/json' \
+                  --data-raw '{
+                     "event": "newCall",
+                     "from": "493023125741",
+                     "to": "492214710334",
+                     "direction": "in",
+                     "callId": "307fa962-de8d-4ffc-817b-7f6993204159",
+                     "user": ["Christopher Miller", "Emma Taylor"]
+                  }'
+
+
+   .. tab:: form-data
+
+      ``POST``-Request sent:
+      ``https://{FQDN-Zammad}/api/v1/cti/{instance specific token}``
+
+      Outbound
+         Payload:
+
+            .. code-block::
+
+               event:"newCall"
+               from:"493023125741"
+               to:"492214710334"
+               direction:"out"
+               callId:"f0871278-0600-4f5c-a746-bec3acf04f41"
+               user:"Christopher Miller"
+
+         Returns:
+
+            .. code-block:: json
+
+               {
+                  "action": "dial",
+                  "caller_id": "496990009111",
+                  "number": "492214710334"
+               }
+
+         Sample curl command:
+         
+            .. code-block:: sh
+
+               $ curl --request POST 'https://{FQDN-Zammad}/api/v1/cti/{instance specific token}' \
+                  --form 'event="newCall"' \
+                  --form 'from="493023125741"' \
+                  --form 'to="492214710334"' \
+                  --form 'direction="out"' \
+                  --form 'callId="f0871278-0600-4f5c-a746-bec3acf04f41"' \
+                  --form 'user="Christopher Miller"'
+
+      Inbound
+         Payload:
+
+            .. code-block::
+
+               event:"newCall"
+               from:"493023125741"
+               to:"492214710334"
+               direction:"in"
+               callId:"25641e3f-3317-4c48-80b3-fc573c7ffe2b"
+               user[]:"Christopher Miller"
+               user[]:"Emma Taylor"
+
+         Returns:
+
+            .. code-block:: json
+
+               {}
+
+         Sample curl command:
+         
+            .. code-block:: sh
+
+               $ curl --request POST 'https://{FQDN-Zammad}/api/v1/cti/{instance specific token}' \
+                  --form 'event="newCall"' \
+                  --form 'from="493023125741"' \
+                  --form 'to="492214710334"' \
+                  --form 'direction="in"' \
+                  --form 'callId="25641e3f-3317-4c48-80b3-fc573c7ffe2b"' \
+                  --form 'user[]="Christopher Miller"' \
+                  --form 'user[]="Emma Taylor"'
+
+   .. tab:: URL variables
+
+      .. include:: /api/generic-cti/generic-cti_no-more-url-varibales.include.rst
+
```

Comment:
Add "Response" header to better outline the context switch. Maybe put the whole section before the example showcase (currently) above.

Reply:
I'm not sure if I can follow here?
Does this count for all 3 sub pages?

As I'm not sure what you're meaning I'm assuming you're talking about me putting the returns below the sample payloads? If so, we're doing that on the whole documentation actually:

https://docs.zammad.org/en/latest/api/user.html#create

Reply:
Only "New call" has a specific section about the response, starting with "Depending on the chosen call direction (...)". My suggestion is to give this section a header/title "Response".

Reply:
I believe that "Response" as a stand alone heading without anything else is slightly unlucky when having the prior coming text in mind:

![image](https://user-images.githubusercontent.com/6549061/156000629-346c5041-2d27-488c-98bc-8421aca78088.png)

If you think a heading is useful at that part, how about something like `Situation specific responses` or similar?

Reply:
Sounds good.